easybuild/easyconfigs/x/xtb/xtb-6.4.1-foss-2021b.eb

```
@@ -0,0 +1,41 @@
+easyblock = 'MesonNinja'
+
+name = 'xtb'
+version = '6.4.1'
+
+homepage = 'https://xtb-docs.readthedocs.io'
+description = """ xtb - An extended tight-binding semi-empirical program package. """
+
+toolchain = {'name': 'foss', 'version': '2021b'}
+
+source_urls = [GITHUB_LOWER_SOURCE]
+sources = [{'download_filename': 'v%(version)s.tar.gz', 'filename': SOURCE_TAR_GZ}]
+checksums = ['cd7b6ec9b7963012ce71220a70773641f0d9e06e0691750a25b83e823510d1d7']
+
+github_account = 'grimme-lab'
+builddependencies = [
+    ('Meson', '0.58.2'),
+    ('Ninja', '1.10.2'),
+]
+
+configopts = "-Dla_backend='custom' "
+configopts += "-Dcustom_libraries='flexiblas' "
+configopts += "--optimization=2 "
+configopts += "--buildtype release "
+configopts += "--warnlevel 0 "
+configopts += "-Dpython=false"
```

```suggestion
configopts = "-Dla_backend='custom' "
configopts += "-Dcustom_libraries='flexiblas' "
configopts += "--buildtype release "
```

Based off the options in the intel toolchain for this version and the following in the output:
```
WARNING: Recommend using either -Dbuildtype or -Doptimization + -Ddebug. Using both is redundant since they override each other. See: https://mesonbuild.com/Builtin-options.html#build-type-options
```
```
WARNING: Unknown options: "python"
```